operator/pkg/certs/certs.go

```diff
@@ -46,7 +46,7 @@ import (
 const (
 	// CertificateBlockType is a possible value for pem.Block.Type.
 	CertificateBlockType = "CERTIFICATE"
-	rsaKeySize           = 2048
+	rsaKeySize           = 3072
```

Comment:
I prefer these two components just have their own global variables.

Currently, these two components have no common dependencies except for the API. I don't want them to have common dependencies coupled together just because of this small tip.

Reply:
`operator` and `karmadactl` are more like two installation methods than two components. And karmada should behave the same in different installations, so I think there could be a common set of base configurations directly for different installations. 
This issue can be put aside for the time being, and if necessary, we can submit a new pr and discuss it later.